Go 1.15: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)

Go 1.15 rc1 on Fedora Rawhide:

```
Testing    in: /builddir/build/BUILD/vultr-2.0.1/_build/src
         PATH: /builddir/build/BUILD/vultr-2.0.1/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/vultr-2.0.1/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/JamesClonk/vultr/version=2.0.1 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/JamesClonk/vultr
github.com/JamesClonk/vultr/lib
# github.com/JamesClonk/vultr/lib
./servers.go:695:17: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./servers.go:696:17: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./servers.go:697:17: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/JamesClonk/vultr/lib [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/lib/servers.go
+++ b/lib/servers.go
@@ -692,9 +692,9 @@ func (c *Client) BackupSetSchedule(id string, bs BackupSchedule) error {
 	values := url.Values{
 		"SUBID":     {id},
 		"cron_type": {bs.CronType},
-		"hour":      {string(bs.Hour)},
-		"dow":       {string(bs.Dow)},
-		"dom":       {string(bs.Dom)},
+		"hour":      {fmt.Sprintf("%v", bs.Hour)},
+		"dow":       {fmt.Sprintf("%v", bs.Dow)},
+		"dom":       {fmt.Sprintf("%v", bs.Dom)},
 	}
 	return c.post(`server/backup_set_schedule`, values, nil)
 }

